homeassistant/components/tibber/sensor.py

```diff
@@ -312,39 +350,30 @@ class TibberSensorRT(TibberSensor):
 
     _attr_should_poll = False
 
-    def __init__(
-        self, tibber_home, sensor_name, device_class, unit, initial_state, state_class
-    ):
+    def __init__(self, tibber_home, metadata: TibberSensorMetadata, initial_state):
         """Initialize the sensor."""
         super().__init__(tibber_home)
-        self._sensor_name = sensor_name
         self._model = "Tibber Pulse"
         self._device_name = f"{self._model} {self._home_name}"
+        self._metadata = metadata
 
-        self._attr_device_class = device_class
-        self._attr_name = f"{self._sensor_name} {self._home_name}"
+        self._attr_device_class = metadata.device_class
+        self._attr_name = f"{metadata.name} {self._home_name}"
         self._attr_state = initial_state
-        self._attr_unique_id = f"{self._tibber_home.home_id}_rt_{self._sensor_name}"
-        self._attr_unit_of_measurement = unit
-        self._attr_state_class = state_class
-        if sensor_name in [
-            "last meter consumption",
-            "last meter production",
-        ]:
+        self._attr_unique_id = f"{self._tibber_home.home_id}_rt_{metadata.name}"
+
+        if metadata.name in ["accumulated cost", "accumulated reward"]:
```

Comment:
```suggestion
        if metadata.name in ("accumulated cost", "accumulated reward"):
```
You can use a tuple here